exporter/otlphttpexporter/otlp.go

```diff
@@ -142,13 +144,9 @@ func (e *baseExporter) export(ctx context.Context, url string, request []byte) e
 	// Format the error message. Use the status if it is present in the response.
 	var formattedErr error
 	if respStatus != nil {
-		formattedErr = fmt.Errorf(
-			"error exporting items, request to %s responded with HTTP Status Code %d, Message=%s, Details=%v",
-			url, resp.StatusCode, respStatus.Message, respStatus.Details)
+		formattedErr = gstatus.ErrorProto(respStatus)
```

Comment:
Do we know that the gRPC code in the response body more accurately describes the error than the HTTP status code returned by the backend?

Reply:
I believe so. The code for readResponse from a few lines above states:

```
		// Request failed. Read the body. OTLP spec says:
		// "Response body for all HTTP 4xx and HTTP 5xx responses MUST be a
		// Protobuf-encoded Status message that describes the problem."
```

Given that our OTLP receiver and exporter are sending and receiving proto payloads, I would say that the gRPC status codes are generally better at describing what happened.